File: javaprog/Maximum.java
import java.util.Scanner;
import java.util.Scanner;
class Main
{
  public static void main(String[] args)
  {
    int Counter, Maximum;
    java.util.Scanner in = new java.util.Scanner(System.in);
    int[] array = new int[11];
    System.out.println("To find the maximum of ten numbers,");
    for(Counter = 1; Counter < array.length; Counter++)
    {
      System.out.print("Enter the " + Counter + " number");
      array[Counter] = in.nextInt();
    }
    Maximum = array[0];
    for(Counter = 0; Counter < array.length; Counter++)
    {
      if(Maximum <= array[Counter])
      {
        Maximum = array[Counter];
      }
    }
    System.out.printf("The maximum of given ten numbers is %d.\n", Maximum);
  }
}

File: javascript/maximum.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
   importPackage(java.io);
   importPackage(java.lang);
   var stdin = new BufferedReader(new InputStreamReader(System['in']));
   return function() {
     return String(stdin.readLine());   
   };                                    
 }());
print("Enter any 10 numbers:");
 Number1 = readln();
var Count = new Array(10);
var ref = "";
//var Max = 0;
Count[1] = Number1;
var Max = Number1;
for(Counter = 2; Counter < 11; Counter++)
{
  print("Enter next numbers:")
  Number1 = readln();
  Count[Counter] = Number1;
  ref += " " + Count[Counter];
}
for(Counter = 1; Counter < 11; Counter++)
{
  if( Max < Count[Counter])
  {
    Max = Count[Counter];
  }
}
print("The maximum number of 10 numbers is " + Max + ".");

